content/articles/domains-jp.md

```diff
@@ -0,0 +1,32 @@
+---
+title: .JP Domains
+excerpt: This article explains the requirements and special procedures for .JP domain names.
+categories:
+- Domains
+---
+
+# .JP Domain Names
+
+* TOC
+{:toc}
+
+---
+
+This article explains the requirements and special procedures for .JP domain names.
+
+
+## Renewing a .JP domain {#renewing}
+
+### Auto-renewal only {#auto-renewal-required}
+
+This TLD doesn't support manual renewals. To successfully renew the domain, the [auto-renewal feature](/articles/domain-auto-renewal) must be enabled at least 1 month before the domain will expire, as explained in the ["auto-renew only" article](/articles/auto-renew-only-domains).
+
+If auto-renewal is not enabled, the domain will be scheduled for removal **12 days before the expiration**. The registrar can deactivate it at any point during this period (even before the expiration date). It is possible to [restore](/articles/restoring-domain) the domain before final deletion, but the registry will charge a redemption fee.
+
+## Transferring a .JP domain away from DNSimple {#transfer-away}
+
+Transfers are NOT possible, if the domain is expiring within the next 30 days.
```

Comment:
Should we also add that a 60-day lock is applied after transfer?

https://kb.centralnicreseller.com/domains/tlds/jp